pkg/cloudprovider/aws/block_storage_adapter.go

```diff
@@ -88,10 +73,10 @@ func NewBlockStorageAdapter(region, availabilityZone string) (cloudprovider.Bloc
 // from snapshot.
 var iopsVolumeTypes = sets.NewString("io1")
 
-func (op *blockStorageAdapter) CreateVolumeFromSnapshot(snapshotID, volumeType string, iops *int64) (volumeID string, err error) {
+func (op *blockStorageAdapter) CreateVolumeFromSnapshot(snapshotID, volumeType, volAZ string, iops *int64) (volumeID string, err error) {
```

Comment:
Perhaps we just fully spell this out to `volumeAZ` similar to `volumeType`

Reply:
Good idea.